<!-- papers.html -->

<!DOCTYPE html>
<html>
<head>
    <title>Publications</title>
    <link rel="stylesheet" href="styles/style.css">
    <!-- Add your CSS and other head elements here -->
</head>
<body class="sub-page">
      <!-- Include the header directly -->
      <header>
        <nav class="header-nav">
            <ul>
                <li><a href="index.html">Home</a></li>
                <li><a href="research.html">Research</a></li>
                <li><a href="publications.html">Publications</a></li>
                <li><a href="protocols.html">Protocols</a></li>
                <li><a href="3D-printing.html">3D-printing</a></li>
                <li><a href="art.html">Art</a></li>
            </ul>
        </nav>
    </header>
    <div class="content">
    <h1>Publications</h1>

    <p>Most up to date papers list on <a href="https://scholar.google.com/citations?user=CfmCibkAAAAJ&hl=en">Google Scholar</a></p>

    <hr> <!-- Add a horizontal line to separate papers -->
    
    <h2>Cryosectioning-enabled super-resolution microscopy for studying nuclear architecture at the single protein level</h2>
    <p>Authors: Johannes Stein, Maria Ericsson, Michel Nofal, Lorenzo Magni, Sarah Aufmkolk, Ryan B. McMillan, <span class="highlight-name">Laura Breimann</span>, Conor P. Herlihy, S. Dean Lee, Andréa Willemin, Jens Wohlmann, Laura Arguedas-Jimenez, Peng Yin, Ana Pombo, George M. Church, Chao-ting Wu</p>
    <p>Date: Feburary 5, 2024</p>
    <p>Journal: bioRxiv</p>
    <p>Abstract: DNA-PAINT combined with total Internal Reflection Fluorescence (TIRF) microscopy enables the highest localization precisions, down to single nanometers in thin biological samples, due to TIRF’s unique method for optical sectioning and attaining high contrast. However, most cellular targets elude the accessible TIRF range close to the cover glass and thus require alternative imaging conditions, affecting resolution and image quality. Here, we address this limitation by applying ultrathin physical cryosectioning in combination with DNA-PAINT. With “tomographic & kinetically-enhanced” DNA-PAINT (tokPAINT), we demonstrate the imaging of nuclear proteins with sub-3 nanometer localization precision, advancing the quantitative study of nuclear organization within fixed cells and mouse tissues at the level of single antibodies. We believe that ultrathin sectioning combined with the versatility and multiplexing capabilities of DNA-PAINT will be a powerful addition to the toolbox of quantitative DNA-based super-resolution microscopy in intracellular structural analyses of proteins, RNA and DNA in situ.</p>
    <p>Link: <a href="https://www.biorxiv.org/content/10.1101/2024.02.05.576943v1.abstract">Read Paper</a></p>
   
    
    <hr> <!-- Add a horizontal line to separate papers -->
    
    


    
    <h2>mRNA stability and m6A are major determinants of subcellular mRNA localization in neurons</h2>
    <p>Authors: Inga Loedige, Artem Baranovskii, Samantha Mendonsa, Sayaka Dantsuji, Niko Popitsch, <span class="highlight-name">Laura Breimann</span>, Nadja Zerna, Vsevolod Cherepanov, Miha Milek, Stefan Ameres, Marina Chekulaeva</p>
    <p>Date: August 3, 2023</p>
    <p>Journal: Molecular Cell</p>
    <p>Abstract: For cells to perform their biological functions, they need to adopt specific shapes and form functionally distinct subcellular compartments. This is achieved in part via an asymmetric distribution of mRNAs within cells. Currently, the main model of mRNA localization involves specific sequences called “zipcodes” that direct mRNAs to their proper locations. However, while thousands of mRNAs localize within cells, only a few zipcodes have been identified, suggesting that additional mechanisms contribute to localization. Here, we assess the role of mRNA stability in localization by combining the isolation of the soma and neurites of mouse primary cortical and mESC-derived neurons, SLAM-seq, m6A-RIP-seq, the perturbation of mRNA destabilization mechanisms, and the analysis of multiple mRNA localization datasets. We show that depletion of mRNA destabilization elements, such as m6A, AU-rich elements, and suboptimal codons, functions as a mechanism that mediates the localization of mRNAs associated with housekeeping functions to neurites in several types of neurons.</p>
    <p>Link: <a href="https://www.sciencedirect.com/science/article/pii/S1097276523004689?via%3Dihub">Read Paper</a></p>
    <p>The image analysis GitHub repo: <a href="https://github.com/LauraBreimann/neuron_Puro-PLA_quantification">GitHub link</a></p>
    
    <hr> <!-- Add a horizontal line to separate papers -->
    
    
    
    
    <h2>Massively parallel identification of mRNA localization elements in primary cortical neurons</h2>
    <p>Authors: Samantha Mendonsa, Nicolai von Kügelgen, Sayaka Dantsuji, Maya Ron, <span class="highlight-name">Laura Breimann</span>, Artem Baranovskii, Inga Lödige, Marieluise Kirchner, Meret Fischer, Nadja Zerna, Lucija Bujanic, Philipp Mertins, Igor Ulitsky & Marina Chekulaeva</p>
    <p>Date: January 16, 2023</p>
    <p>Journal: Nature Neuroscience</p>
    <p>Abstract: Cells adopt highly polarized shapes and form distinct subcellular compartments in many cases due to the localization of many mRNAs to specific areas, where they are translated into proteins with local functions. This mRNA localization is mediated by specific cis-regulatory elements in mRNAs, commonly called ‘zipcodes’. Although there are hundreds of localized mRNAs, only a few zipcodes have been characterized. Here we describe a novel neuronal zipcode identification protocol (N-zip) that can identify zipcodes across hundreds of 3′ untranslated regions. This approach combines a method of separating the principal subcellular compartments of neurons—cell bodies and neurites—with a massively parallel reporter assay. N-zip identifies the let-7 binding site and (AU)n motif as de novo zipcodes in mouse primary cortical neurons. Our analysis also provides, to our knowledge, the first demonstration of an miRNA affecting mRNA localization and suggests a strategy for detecting many more zipcodes.</p>
    <p>Link: <a href="https://www.nature.com/articles/s41593-022-01243-x">Read Paper</a></p>
    <p>The image analysis GitHub repo: <a href="https://github.com/LauraBreimann/smFISH_neuron_analysis">GitHub link</a></p>
    
    
    <hr> <!-- Add a horizontal line to separate papers -->
   
   
   
    <h2>RS-FISH: precise, interactive, fast, and scalable FISH spot detection</h2>
    <p>Authors: Ella Bahry *, <span class="highlight-name">Laura Breimann</span> *, Marwan Zouinkhi *, Leo Epstein, Klim Kolyvanov, Nicholas Mamrak, Benjamin King, Xi Long, Kyle I S Harrington, Timothée Lionnet, Stephan Preibisch </p>
    <p>Date: December 19, 2022</p>
    <p>Journal: Nature Methods</p>
    <p>Abstract: Fluorescent in-situ hybridization (FISH)-based methods extract spatially resolved genetic and epigenetic information from biological samples by detecting fluorescent spots in microscopy images, an often challenging task. We present Radial Symmetry-FISH (RS-FISH), an accurate, fast, and user-friendly software for spot detection in two- and three-dimensional images. RS-FISH offers interactive parameter tuning and readily scales to large datasets and image volumes of cleared or expanded samples using distributed processing on workstations, clusters, or the cloud. RS-FISH maintains high detection accuracy and low localization error across a wide range of signal-to-noise ratios, a key feature for single-molecule FISH, spatial transcriptomics, or spatial genomics applications.</p>
    <p>Link: <a href="https://pubmed.ncbi.nlm.nih.gov/36396787/">Read Paper</a></p>
    <p>The software GitHub repo: <a href="https://github.com/PreibischLab/RS-FISH">GitHub link</a></p>
    <img src="/img/RSFISH.jpeg" alt="Embryo with RS-FISH detected smFISH spots">
    
    <hr> <!-- Add a horizontal line to separate papers -->
    
  
    <h2>The histone H4 lysine 20 demethylase DPY-21 regulates the dynamics of condensin DC binding</h2>
    <p>Authors: <span class="highlight-name">Laura Breimann</span>*#, Ana Karina Morao*, Jun Kim, David Sebastian Jimenez, Nina Maryn, Krishna Bikkasani, Michael J. Carrozza, Sarah E. Albritton, Maxwell Kramer, Lena Annika Street, Kustrim Cerimi, Vic-Fabienne Schumann, Ella Bahry, Stephan Preibisch, Andrew Woehler, and Sevinç Ercan#</p>
    <p>Date: January 15, 2022</p>
    <p>Journal: Journal of Cell Science</p>
    <p>Abstract: Condensin is a multi-subunit structural maintenance of chromosomes (SMC) complex that binds to and compacts chromosomes. Here, we addressed the regulation of condensin binding dynamics using Caenorhabditis elegans condensin DC, which represses X chromosomes in hermaphrodites for dosage compensation. We established fluorescence recovery after photobleaching (FRAP) using the SMC4 homolog DPY-27 and showed that a well-characterized ATPase mutation abolishes DPY-27 binding to X chromosomes. Next, we performed FRAP in the background of several chromatin modifier mutants that cause varying degrees of X chromosome derepression. The greatest effect was in a null mutant of the H4K20me2 demethylase DPY-21, where the mobile fraction of condensin DC reduced from ∼30% to 10%. In contrast, a catalytic mutant of dpy-21 did not regulate condensin DC mobility. Hi-C sequencing data from the dpy-21 null mutant showed little change compared to wild-type data, uncoupling Hi-C-measured long-range DNA contacts from transcriptional repression of the X chromosomes. Taken together, our results indicate that DPY-21 has a non-catalytic role in regulating the dynamics of condensin DC binding, which is important for transcription repression.
    </p>
    <p>Link: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8917352/">Read Paper</a></p>
    <p>The analysis GitHub repo: <a href="https://github.com/ercanlab/2021_Breimann_et_al">GitHub link</a></p>
    <img src="/img/FRAP.jpg" alt="FRAP curves of DPY-27">
    
    <hr> <!-- Add a horizontal line to separate papers -->

    
    <h2>Light-microscopy methods in C. elegans research</h2>
    <p>Authors: <span class="highlight-name">Laura Breimann</span>*, Friedrich Preusser*, Stephan Preibisch</p>
    <p>Date: Feburary 1, 2019</p>
    <p>Journal: Current Opinion in Systems Biology</p>
    <p>Abstract: Ever since Caenorhabditis elegans was introduced as a model system it has been tightly linked to microscopy, which has led to significant advances in understanding biology over the last decades. Developing new technologies therefore is an essential part in the endeavor to gain further mechanistic insights into developmental biology. This review will discuss state-of-the-art developments in quantitative light microscopy in the context of C. elegans research as well as the impact these technologies have on the field. We will highlight future developments that currently promise to revolutionize biological research by combining sequencing-based single-cell technologies with high-resolution quantitative imaging.</p>
    <p>Link: <a href="https://www.sciencedirect.com/science/article/pii/S2452310018300842">Read Paper</a></p>
    <img src="/img/1-s2.0-S2452310018300842-fx1_lrg.jpg" alt="Graphical Abstract">

    <hr> <!-- Add a horizontal line to separate papers -->



    <h2>DamID profiling of dynamic Polycomb-binding sites in Drosophila imaginal disc development and tumorigenesis</h2>
    <p>Authors: Marco La Fortezza, Giovanna Grigolon, Andrea Cosolo, Alexey Pindyurin, <span class="highlight-name">Laura Breimann</span>, Helmut Blum, Bas van Steensel & Anne-Kathrin Classen</p>
    <p>Date: June 5, 2018</p>
    <p>Journal: Epigenetics & Chromatin</p>
    <p>Abstract: Background: Tracking dynamic protein–chromatin interactions in vivo is key to unravel transcriptional and epige‑netic transitions in development and disease. However, limited availability and heterogeneous tissue composition ofin vivo source material impose challenges on many experimental approaches.Results: Here we adapt cell‑type‑specific DamID‑seq profiling for use in Drosophila imaginal discs and make FLP/FRT‑based induction accessible to GAL driver‑mediated targeting of specific cell lineages. In a proof‑of‑principleapproach, we utilize ubiquitous DamID expression to describe dynamic transitions of Polycomb‑binding sites duringwing imaginal disc development and in a scrib tumorigenesis model. We identify Atf3 and Ets21C as novel Polycombtarget genes involved in scrib tumorigenesis and suggest that target gene regulation by Atf3 and AP‑1 transcriptionfactors, as well as modulation of insulator function, plays crucial roles in dynamic Polycomb‑binding at target sites. Weestablish these findings by DamID‑seq analysis of wing imaginal disc samples derived from 10 larvae.Conclusions: Our study opens avenues for robust profiling of small cell population in imaginal discs in vivo and pro‑vides insights into epigenetic changes underlying transcriptional responses to tumorigenic transformation.</p>
    <p>Link: <a href="https://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-018-0196-y">Read Paper</a></p>

    <hr> <!-- Add a horizontal line to separate papers -->


    
</div>
</body>
</html>

